### ISSUE TITLE
process is not defined

### ISSUE THREAD

**Issue by CRC32EX** [ghostdevv/rollup-obfuscator#108]
# Steps to reproduce this issue

1. Init Vue3 project
```
npm init vue@latest
```
![image](https://user-images.githubusercontent.com/25629617/160607472-94954216-5b1b-4c96-93a4-7dee9f195140.png)

2. Install modules
```
cd vue-project
npm install
npm install -D rollup-obfuscator javascript-obfuscator
```

3. Edit vite.config.ts
```
vim vite.config.ts
```
![image](https://user-images.githubusercontent.com/25629617/160607336-a52866c4-0303-40a1-9907-44269b2c3a17.png)

4. Build
```
npm run build
```

5. Start local webserver
```
cd vue-project/dist
python -m http.server 8000
```

6. Open URL
```
http://127.0.0.1:8000
```

7. Error
```
Uncaught ReferenceError: process is not defined
    at index.64907874.js:1:11585
```

![image](https://user-images.githubusercontent.com/25629617/161315317-c4d11d86-f443-4adc-8463-4af078f95f9f.png)

**Comment by ghostdevv** [comment]
If you remove the plugin is process still undefined? @CRC32EX

**Comment by CRC32EX** [comment]
![image](https://user-images.githubusercontent.com/25629617/161361254-bdf98b3b-9ed8-469d-9790-70f44b56b87b.png)
![image](https://user-images.githubusercontent.com/25629617/161361347-cf23560e-2f63-4dc8-b977-f0418e33789f.png)

**Comment by ghostdevv** [comment]
Been looking into it and seems to only effect vue, will continue to try and figure it out. Unfortunately setting the target of the obfuscator to browser didn't fix it

**Comment by ghostdevv** [comment]
@CRC32EX This is fixed in `2.1.2`
